app/src/main/res/values-sw600dp-land/dimens.xml

```diff
@@ -445,5 +445,9 @@
 
   <!-- OptionAudioLanguage -->
   <dimen name="option_audio_language_padding_start">16dp</dimen>
-  <dimen name="option_audio_language_padding_end">16dp</dimen>
-</resources>
+  <dimen name="option_audio_language_padding_end">16dp</dimen>\
```

Comment:
There is `\` at the end. This is the reason for all the failures as the app is unable to build.

Reply:
Sorry, I couldn't identify this earlier